cms/db/contest.py

```diff
@@ -99,6 +99,12 @@ class Contest(Base):
         nullable=False,
         default=True)
 
+    # Allow unofficial submission after participation ended
+    allow_unofficial_submission_after_participation_ended = Column(
```

Comment:
The naming here is a bit misleading because even after contest ends, the participation is actually still not "ended" (it is still in the database).

I think a more accurate naming could be

```
allow_unofficial_submission_before_analysis_mode
```

What do you think?